lib/pbench/test/unit/server/query_apis/commons.py

```diff
@@ -224,18 +224,11 @@ def test_accessing_user_data_with_invalid_token(
             pytest.skip(
                 "skipping " + self.test_accessing_user_data_with_invalid_token.__name__
             )
-        # valid token logout
-        response = self.make_request_call(
-            client,
-            server_config.rest_uri + "/logout",
-            {"Authorization": "Bearer " + pbench_token},
-        )
-        assert response.status_code == HTTPStatus.OK
         self.payload[userp.parameter.name] = user
         response = self.make_request_call(
             client,
             server_config.rest_uri + self.pbench_endpoint,
-            {"Authorization": "Bearer " + pbench_token},
+            {"Authorization": "Bearer " + pbench_token_invalid},
```

Comment:
Does the comment at line 211 now need updating?

Reply:
No, because the nature of the token hasn't changed as we are still providing an expired token. The only difference is that instead of logging out the valid token and then using it, we are now providing the already-expired token directly.